corehq/apps/domain/models.py

```diff
@@ -1047,6 +1059,23 @@ def update_calculations(cls, domain, property_to_update):
         obj.save(update_fields=[property_to_update])
 
 
+class AllowedDTEExpressionSettings(models.Model):
+    """
+    Model contains DTE(aka UCR) expressions settings for a domain.
```

Comment:
In the beginning, I had a similar question in my mind, whether to go with UCR or with DTE. The reasoning behind going with this name is that we are planning to rename UCR with Data Transformation Engine. The cosmetic changes have started with this PR. 
I was not sure if we would rename it in code in near future but that seemed plausible so I went with DTE.

Now I did not think about DET(Data Export Tool) before. It can be confusing. Now I am also inclined to go with UCRs instead of DET, just want to confirm it would be a good idea.

Reply:
I think sticking with UCR for now makes sense. Changing all instances to DTE is a big job and not clear what the value is.

Reply:
I have renamed DTE to UCR.